gostream/codec/h264/encoder.go

```diff
@@ -66,7 +66,7 @@ func NewEncoder(width, height, keyFrameInterval int, logger golog.Logger) (codec
 	}
 
 	if h.frame = avutil.FrameAlloc(); h.frame == nil {
-		h.Close()
+		h.Close() //nolint:errcheck
```

Comment:
Not sur eif this was your preference or if you were unaware of the other option, "goutils.UncheckedError":
```
components/arm/eva/eva.go:              utils.UncheckedError(res.Body.Close())
```

But if it was up to me, we'd just suppress the linter directly like you have done.

Reply:
wasn't aware, useful

was planning to hand these off to code owners, nolint is just a stopgap so we can restore the checks